spark-fabric/src/main/java/me/lucko/spark/fabric/FabricWorldInfoProvider.java

```diff
@@ -140,10 +134,16 @@ public CountsResult pollCounts() {
                 return null;
             }
 
-            ClientEntityManager<Entity> entityManager = ((ClientWorldAccessor) world).getEntityManager();
-            EntityIndex<?> entityIndex = ((ClientEntityManagerAccessor) entityManager).getIndex();
+            int entities;
+
+            if (FabricLoader.getInstance().isModLoaded("moonrise")) {
```

Comment:
Instead of this check, it might be better if we just check for the presence of the getEntityCount method in a static initialiser block, then just use the method if it exists?

Reply:
Moonrise extends the entity getter with its own implementation, it doesn't add the method to the vanilla one. Se we need an instance in order to look up the method without hardcoding the Moonrise class name as well.

Reply:
Ah ok, makes sense